include/services/collection.h

```diff
@@ -228,7 +228,7 @@ class Collection
     bool resize(size_t newCapacity)
     {
         if(newCapacity <= _capacity) { return true; }
-        T *newArray = (T *)services::daal_malloc(sizeof(T) * newCapacity);
+        T *newArray = (T *)services::daal_calloc(sizeof(T) * newCapacity);
```

Comment:
does it affect performance?

Reply:
This changes were checked in some algorithms and performance is ok, but full performance results might be different.